neptune/new/internal/backends/hosted_neptune_backend.py

```diff
@@ -184,11 +193,18 @@ def __init__(self, credentials: Credentials, proxies: Optional[Dict[str, str]] =
             python_version=platform.python_version())
         self._http_client.session.headers.update({'User-Agent': user_agent})
 
+        if self.missing_features:
+            warnings.warn(NeptuneFeaturesNotAvailableWarning(self.missing_features).message)
```

Comment:
I am not a fan of printing this in every single Run.

Reply:
it's (now) a python warning, so the use can setup it to print only once
`warnings.simplefilter("once", NeptuneFeaturesNotAvailableWarning)`
or choose to ignore it alltogether
`warnings.simplefilter("ignore", NeptuneFeaturesNotAvailableWarning)`

either way, your fight is with spec, not me ;)

Reply:
spec has changed, so removing